src/resources/projects/website/listing/item-grid.ejs.md

```diff
@@ -37,83 +37,101 @@ return !["title", "image", "image-alt", "date", "author", "subtitle", "descripti
 });
 %>
 
-<div class="g-col-1" <%= listing.utilities.metadataAttrs(item) %>>
+::: {.g-col-1 <%= listing.utilities.metadataAttrs(item) %> }
```

Comment:
Same as for the other file.
Using div syntax to avoid parsing issues with the use of. `` ` `` either in raw inline or block.